srcpkgs/66/template

```diff
@@ -0,0 +1,37 @@
+# Template file for '66'
+pkgname=66
+version=0.1.0.0
+revision=1
+wrksrc="66-v${version}"
+build_style=gnu-configure
+configure_args="--with-sysdeps=${XBPS_CROSS_BASE}/usr/lib/skalibs/sysdeps
+ --with-lib=${XBPS_CROSS_BASE}/usr/lib/oblibs
```

Comment:
2 calls to --with-lib ?

Reply:
Needed when using default dirs. Removed.